### PR TITLE
release: v1.15 release notes

### PR DESCRIPTION
**Pull request by mrpollo**
First pass on release notes for the upcoming PX4 v1.15

**Next Steps**
* Upgrade notice and guide
* Make sure all major features are listed

### CODE REVIEW

**Comment by hamishwillee** [comment]
Looks like you're getting somewhere. When this goes in I will push to the v1.15 branch. 

It might be worth adding a section on changed parameters. I have a script that does this, but unfortunately it relies on this issue being fixed https://github.com/PX4/PX4-Autopilot/issues/23442

**Comment by mrpollo** [comment]
I went through the whole changelog and picked what I thought was worthwhile. It still needs review with some of the developers. I plan to do that on the next dev call. Then, we probably want to standardize some of the stylized elements across the list so they look uniform across the whole thing.

I'll look at the missing parameters from your script and try to fix them.

**Comment by hamishwillee** [comment]
> I'll look at the missing parameters from your script and try to fix them.

**NOTE** This may actually be a pretty big deal_. The JSON  data is the source for the metadata used by QGC and compressed/encoded on device for the metadata toolchain - https://docs.px4.io/main/en/advanced/px4_metadata.html#metadata-toolchain.

So this also breaks automatic and correct parameter metadata on the vehicle - so we'll start getting those warnings about QGC parameter mismatches again. Not good.

**Comment by hamishwillee** [comment]
@bkueng Fixed this in https://github.com/PX4/PX4-Autopilot/pull/23464 . We should IMO backport it - I cherry-picked the PR in https://github.com/PX4/PX4-Autopilot/pull/23505

**Comment by mrpollo** [comment]
That was quick! thank you both @bkueng @hamishwillee

File: en/releases/1.15.md [hunk 1]
```diff
@@ -17,34 +17,71 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
...
+- Support for a remote parameter database. [PX4-Autopilot#22836](https://github.com/PX4/PX4-Autopilot/pull/22836)
+- New platform independent UART interface. [PX4-Autopilot#21723](https://github.com/PX4/PX4-Autopilot/pull/21723)
 
 ### Control
 
-- [offboard][ros2 offboard control](../flight_modes/offboard.md#ros-2-messages) allows for direct motors and servo control.
+- [offboard] [ros2 offboard control](../flight_modes/offboard.md#ros-2-messages) allows for direct motors and servo control.
   Added in [PX4-Autopilot#22222](https://github.com/PX4/PX4-Autopilot/pull/22222).
 
 ### Estimation
```
Comment (mrpollo) [inline en/releases/1.15.md:65]: @bresch please see below, I went trough the changelog without seeing your discord message which I will be incorporating into this. 

For now can you do a quick review here please? inline comments are ok

File: en/releases/1.15.md [hunk 2]
```diff
@@ -17,34 +17,71 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
...
 ### Hardware Support
 
-- TBD
+- [Pixhawk Standards] Pixhawk Autopilot Bus (PAB) Split Base ID and FMUM ID [PX4-Autopilot#22639](https://github.com/PX4/PX4-Autopilot/pull/22639)
+- [New Hardware] Support for FMUv6XRT - Pixhawk 6X-RT [PX4-Autopilot#22263](https://github.com/PX4/PX4-Autopilot/pull/22263)
+- [New Hardware] MicoAir H743 [PX4-Autopilot#23218](https://github.com/PX4/PX4-Autopilot/pull/23218)
+- 3DR Control Zero Classic Support Improvements [PX4-Autopilot#22971](https://github.com/PX4/PX4-Autopilot/pull/22971)
+- [New Hardware] ARK Mosaic-X5 RTK GPS [PX4-Autopilot#22735](https://github.com/PX4/PX4-Autopilot/pull/22735)
+- [New Hardware] ARK Pi6X Flow [PX4-Autopilot#22829](https://github.com/PX4/PX4-Autopilot/pull/22829)
+- Refactor uncommanded descend Quad-Chute. [PX4-Autopilot#21598](https://github.com/PX4/PX4-Autopilot/pull/21598)
```
Comment (bkueng) [inline en/releases/1.15.md:35]: This should be under VTOL

File: en/releases/1.15.md [hunk 3]
```diff
@@ -17,34 +17,71 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
...
+  New parameter: VT_QC_ALT_LOSS
+- [FMUv6X]: Increase TELEM2 RX buffer size to increase throughput. [PX4-Autopilot#21660](https://github.com/PX4/PX4-Autopilot/pull/21660)
+- Feature Add power module selector for Auterion power modules. [PX4-Autopilot#22073](https://github.com/PX4/PX4-Autopilot/pull/22073)
```
Comment (bkueng) [inline en/releases/1.15.md:38]:
```suggestion
- Feature: Add power module selector for Auterion power modules. [PX4-Autopilot#22073](https://github.com/PX4/PX4-Autopilot/pull/22073)
```

File: en/releases/1.15.md [hunk 4]
```diff
@@ -60,49 +97,79 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
   Added to PX4 in [PX4-Autopilot#22467](https://github.com/PX4/PX4-Autopilot/pull/22467).
 - [JMAVSim Simulator](../sim_jmavsim/index.md) is now [community supported](../simulation/community_supported_simulators.md).
   It has been deprecated as part of the official PX4 development toolchain in order to focus our efforts on [Gazebo](../sim_gazebo_gz/index.md).
+- [Tools/simulation/gz]: load ApplyLinkWrench plugin by default [PX4-Autopilot#21874](https://github.com/PX4/PX4-Autopilot/pull/21874)
+- Allow user to specify Gazebo Garden/Harmony render engine in simulations. [PX4-Autopilot#22693](https://github.com/PX4/PX4-Autopilot/pull/22693)
+- Add support for quadtailsitter in SITL Gazebo. [PX4-Autopilot#20558](https://github.com/PX4/PX4-Autopilot/pull/20558)
+- Feature Parameter for enabling/disabling PX4's internal battery simulator in SITL. [PX4-Autopilot#22710](https://github.com/PX4/PX4-Autopilot/pull/22710)
```
Comment (bkueng) [inline en/releases/1.15.md:103]:
```suggestion
- Feature: Parameter for enabling/disabling PX4's internal battery simulator in SITL. [PX4-Autopilot#22710](https://github.com/PX4/PX4-Autopilot/pull/22710)
```

**Review by bkueng — COMMENTED** [review]
Nice, thanks for going through this.

File: en/releases/1.15.md [hunk 5]
```diff
@@ -17,34 +17,71 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
...
 ### Control
 
-- [offboard][ros2 offboard control](../flight_modes/offboard.md#ros-2-messages) allows for direct motors and servo control.
+- [offboard] [ros2 offboard control](../flight_modes/offboard.md#ros-2-messages) allows for direct motors and servo control.
   Added in [PX4-Autopilot#22222](https://github.com/PX4/PX4-Autopilot/pull/22222).
 
 ### Estimation
 
-- TBD
+- [EKF2] Use Symforce Rot3 and Quaternion operations [PX4-Autopilot#21359](https://github.com/PX4/PX4-Autopilot/pull/21359)
```
Comment (bresch) [inline en/releases/1.15.md:67]: That's already outdated as we now have an error-state EKF in 1.15

File: en/releases/1.15.md [hunk 6]
```diff
@@ -17,34 +17,71 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
...
+- [EKF2] Migrate params to YAML [PX4-Autopilot#22982](https://github.com/PX4/PX4-Autopilot/pull/22982)
+- [EKF2] New parameter: EKF2_DELAY_MAX - Defines the delay between the current time and delayed-time horizon. [PX4-Autopilot#22983](https://github.com/PX4/PX4-Autopilot/pull/22983)
+- [EKF2] Improve heading convergence when no mag is used [PX4-Autopilot#21691](https://github.com/PX4/PX4-Autopilot/pull/21691)
```
Comment (bresch) [inline en/releases/1.15.md:70]:
```suggestion
- [EKF2] Improve heading and gyro bias convergence when no mag is used [PX4-Autopilot#21691](https://github.com/PX4/PX4-Autopilot/pull/21691)
```
(I also edited the description in the PR)

File: en/releases/1.15.md [hunk 7]
```diff
@@ -60,49 +95,80 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
...
   To be able to arm without an airspeed sensor set `SYS_HAS_NUM_ASPD` to 0.
   To not use the airspeed sensor data in the controller, set `FW_USE_AIRSPD` to 0.
   Added to PX4 in [PX4-Autopilot#22510](https://github.com/PX4/PX4-Autopilot/pull/22510).
+- Improvement: Fixed-wing high (dynamic) wind hardening [PX4-Autopilot#21764](https://github.com/PX4/PX4-Autopilot/pull/21764)
+- Attitude controller cleanup [PX4-Autopilot#22472](https://github.com/PX4/PX4-Autopilot/pull/22472)
+- Feature: FW TECS: rename FW_T_THR_DAMP to FW_T_THR_DAMPING and FW_T_I_GAIN_THR to FW_T_THR_INTEG as the interpretation of these parameters has changed. [PX4-Autopilot#22548](https://github.com/PX4/PX4-Autopilot/pull/22548)
+- Feature: Enable high rate logging for VTOL [PX4-Autopilot#22700)](https://github.com/PX4/PX4-Autopilot/pull/22700)
+- Feature: Add functionality to disable rate control of certain axes in fixed-wing Acro. [PX4-Autopilot#21566](https://github.com/PX4/PX4-Autopilot/pull/21566)
+  New parameter: FW_ACRO_AXS_CTRL
+- Add Fixedwing path following as a separate controller state. [PX4-Autopilot#21376](https://github.com/PX4/PX4-Autopilot/pull/21376)
```
Comment (hamishwillee) [inline en/releases/1.15.md:166]:
```suggestion
- Add Fixed-wing path following as a separate controller state. [PX4-Autopilot#21376](https://github.com/PX4/PX4-Autopilot/pull/21376)
```

File: en/releases/1.15.md [hunk 8]
```diff
@@ -17,34 +17,69 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 
 ## Major Changes
 
-- TBD
+- [New] Throw Mode [PX4-Autopilot#21170](https://github.com/PX4/PX4-Autopilot/pull/21170)
+- Improved Optical Flow Support [PX4-Autopilot#21769](https://github.com/PX4/PX4-Autopilot/pull/21769)
```
Comment (hamishwillee) [inline en/releases/1.15.md:21]: @mrpollo Content looks great. At the end we should have a consistency update for how we present the autopilot link as many are different.. I prefer:

```suggestion
- [New] Throw Mode ([PX4-Autopilot#21170](https://github.com/PX4/PX4-Autopilot/pull/21170)).
- Improved Optical Flow Support ([PX4-Autopilot#21769](https://github.com/PX4/PX4-Autopilot/pull/21769))
```

File: en/releases/1.15.md [hunk 9]
```diff
@@ -60,49 +95,80 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
...
 ### Ethernet
 
 - [Ethernet](../advanced_config/ethernet_setup.md): The default static IP address (used as a fallback for DHCP) has changed from `192.168.0.3` to `10.41.10.2`.
   This avoids potential conflicts where the`192.168.0.x` range is used by routers for WIFI/ethernet networks.
   Changed in PX4 in [PX4-Autopilot#22517](https://github.com/PX4/PX4-Autopilot/pull/22517).
 
-### uXRCE-DDS / ROS2
+### MAVLink
+
+- gimbal: fix auto RC and MAVLink mode. [PX4-Autopilot#23236](https://github.com/PX4/PX4-Autopilot/pull/23236)
```
Comment (hamishwillee) [inline en/releases/1.15.md:113]:
```suggestion
- Gimbal: fix auto RC and MAVLink mode. [PX4-Autopilot#23236](https://github.com/PX4/PX4-Autopilot/pull/23236)
```

**Pull request merged**